packages/perspective-viewer-d3fc/src/js/legend/legend.js

```diff
@@ -47,6 +49,11 @@ function legendComponent(scrollLegend, scaleModifier) {
     let scale = null;
     let color = null;
     let draggable = draggableComponent();
+    let resizable = resizableComponent()
+        .zIndex(3)
```

Comment:
As these are the default property values, I don't think we should set them again.

Reply:
Sure, that makes sense. I wanted it to be explicit in case the component ended up be used in other places and those defaults were changed because they stopped making sense, but I suppose that bridge'll be crossed as and when.